packages/browser/src/plugins/segmentio/batched-dispatcher.ts

```diff
@@ -65,7 +68,7 @@ export default function batch(
       const { sentAt, ...newEvent } = event as SegmentEvent
       return newEvent
     })
-
+    console.log('sending batch', updatedBatch)
```

Comment:
Nit: probably want to remove this. Rather than logging them here -- we want to limit console spam since it ends up muddying the customer's rollbar logs (these are logged in production). 

Being able to listen to this via the event emitter would probably be for the best.